jxl/src/frame.rs

```diff
@@ -209,18 +209,26 @@ mod test_frame {
 
     use super::Frame;
 
-    fn read_frame(image: &[u8]) -> Result<Frame, Error> {
+    fn read_frames(image: &[u8]) -> Result<Vec<Frame>, Error> {
         let codestream = ContainerParser::collect_codestream(image).unwrap();
         let mut br = BitReader::new(&codestream);
         let file_header = FileHeader::read(&mut br).unwrap();
-        let mut result = Frame::new(&mut br, &file_header)?;
-        result.decode_lf_global(&mut br)?;
-        Ok(result)
+        let mut frames = vec![];
+        loop {
+            let mut frame = Frame::new(&mut br, &file_header)?;
+            let is_last = frame.is_last();
+            frame.decode_lf_global(&mut br)?;
```

Comment:
TODO() read / skip rest of frame?

Reply:
How to to that exactly? How many bits to skip?

Reply:
I am not sure it's very easy, which is why I'd make it a Todo :-)

Reply:
added todo, but without this, it will just fail on the three images with multiple frames that we have...